guide/auto-update.md

```diff
@@ -6,17 +6,18 @@ outline: deep
 # Automatic reload
 
 With this behavior, once the browser detects a new version of your application, then, it will update the caches and will reload any browser windows/tabs with the application opened automatically to take the control.
-
-::: warning
-In order to reload all client tab/window, you will need to import any virtual module provided by the plugin: if you're not using any virtual, there is no way to interact with the application ui, and so, any client tab/window will not be reloaded (the old service worker will be still controlling the application).
-
-Automatic reload is not automatic page reload, you will need to use the following code in your application entry point if you want **automatic page reload**:
+You will need to use the following code in your application entry point:
 
 ```js
 import { registerSW } from 'virtual:pwa-register'
 
-registerSW({ immediate: true })
```

Comment:
Another bold move because the flag is missleading and has nothing to do with the automatic reload itself but is only used in a very rare case.

Reply:
Question is, should we mention that flag anywhere?